Source: me@gmail.com
Upstream: No
Reason: Not override bindir setting, use external enet lib.
--- supertuxkart-0.6.2/src/Makefile.am~	2009-09-16 01:18:54.467171860 +0800
+++ supertuxkart-0.6.2/src/Makefile.am	2009-09-16 01:19:13.856748112 +0800
@@ -2,11 +2,10 @@
 
 SUBDIRS = ide
 
-bindir=$(prefix)/games
 bin_PROGRAMS = supertuxkart
 
 AM_CPPFLAGS = -DSUPERTUXKART_DATADIR="\"$(datadir)/games/$(PACKAGE)/\""  \
- -I$(srcdir)/bullet/src/ -I$(srcdir)/enet/include/
+ -I$(srcdir)/bullet/src/
 
 supertuxkart_SOURCES = \
  main.cpp \
--- supertuxkart-0.6.2/configure.ac~	2009-09-16 01:28:55.491633114 +0800
+++ supertuxkart-0.6.2/configure.ac	2009-09-16 01:26:20.674599050 +0800
@@ -357,7 +357,6 @@
   ;;
 esac
 SUMMARY="$SUMMARY\nUsing enet network support."
-ENETTREE="src/enet"
 
 # Check for gettext
 CHECK_GETTEXT(external)
@@ -394,7 +393,6 @@
 AC_SUBST(glut_LIBS)
 AC_SUBST(sdl_LIBS)
 AC_SUBST(BULLETTREE)
-AC_SUBST(ENETTREE)
 
 dnl ================
 dnl Create makefiles
@@ -417,8 +415,7 @@
  src/ide/Makefile                   \
  src/bullet/Makefile                \
  src/bullet/src/Makefile            \
- src/bullet/Demos/OpenGL/Makefile   \
- src/enet/Makefile                  \
+ src/bullet/Demos/OpenGL/Makefile
 ])
 AC_OUTPUT
 echo -e $SUMMARY
